pages/register.vue

```diff
@@ -99,7 +99,11 @@ export default {
       this.$router.push(`/${this.profile.username}`)
     },
     async submit({ email, username, password, place }) {
-      if (!email.trim() || !username.trim() || !password.trim()) {
+      if (
+        !(typeof email !== 'string' ? '' : email.trim()) ||
```

Comment:
Maybe something like:


```
 !email || !email.trim() || ...
```

It's a bit shorter and could also work, what do you think?

Reply:
yes, it is a lot more efficient